### ISSUE TITLE
PTS (High Isle) error

### ISSUE THREAD

**Issue by Lyhtande** [Rhyono/CraftStore#12]
On PTS (High Isle) this error occurs:
```lua
user:/AddOns/CraftStoreFixedAndImproved/CraftStore.lua:811: function expected instead of nil
stack traceback:
user:/AddOns/CraftStoreFixedAndImproved/CraftStore.lua:811: in function 'CS.UpdateStudyLine'
<Locals> control = ud, tracking = [table:1]{1 = T}, craft = 1, line = 1, trackingTable = [table:2]{1 = T}, allTracked = T, x = 1, subcontrol = ud </Locals>
user:/AddOns/CraftStoreFixedAndImproved/CraftStore.lua:834: in function 'CS.UpdateAllStudies'
<Locals> craft = 1, craftData = [table:3]{}, line = 1, lineData = [table:1] </Locals>
user:/AddOns/CraftStoreFixedAndImproved/CraftStore_Events.lua:179: in function 'CS.OnPlayerActivated'
<Locals> eventCode = 589824, initial = F </Locals>
```

**Comment by MuMuQ** [comment]
This should be fixed with #13 :)

**Comment by Lyhtande** [comment]
> This should be fixed with #13 :)

It works - thank you!
